CONTRIBUTING.md

```diff
@@ -20,31 +20,28 @@ brew install git
 brew install python3
 ````
 
-## Install Node
-We are using versions: node v7.9.0
-
+## Install Yarn
 ````
-brew install node
+brew install yarn --without-node
 ````
 
-## Install n
+## Install NVM
+
 ````
-sudo npm install -g n
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.6/install.sh | bash
 ````
 
-## Install Yarn
+## Install Node
+We are using node v9.0.0
 
 ````
-npm install -g yarn
+nvm install 9.0.0
 ````
 
 ## Install Node Modules and Global Dependencies
 ````
 yarn
-yarn global add jest@v16.0.2
 yarn global add nodemon
-yarn global add webpack@v2.1.0-beta.25
-yarn global add body-parser
```

Comment:
Just out of curiosity, could you maybe elaborate on the reasoning for these documentation updates (i.e. adding/removing certain commands)? Thanks!

Reply:
Sure, we could add in webpack and body-parser as global installs. That might have been a mistake on my part to take them out. My thought was that we made the documentation when we were still using npm for everything, so I was trying to change the docs to 1) follow the recommended install guides for yarn and nvm and 2) match the linux part of the docs.

Reply:
Oh good point haha. Yeah we don't want those 3 installed globally.